src/components/Missions.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+import Mission from './Mission';
+import './Missions.css';
+import { setMissions } from '../redux/reducers/MissionsReducer';
+
+const Missions = () => {
+  const missions = useSelector((state) => state.missionsReducer);
+  const dispatch = useDispatch();
+  console.log(missions);
```

Comment:
- [ ]  Kindly remove `console.log` statements from your file to make your code cleaner and neater.

Reply:
okii fixed it :)